ci/docker/runtime_functions.sh

```diff
@@ -31,14 +31,49 @@ clean_repo() {
     git submodule update --init --recursive
 }
 
+# wrap compiler calls with ccache
+build_ccache_wrappers() {
+    set -ex
+
+    rm -f cc
+    rm -f cxx
```

Comment:
does it need to ignore remove error?

Reply:
The file might not exist and that's fine. So we don't want to fail